### DISCUSSION TITLE
Censoring reviewer names.

### DISCUSSION THREAD

**Discussion by aghaeian** [bagisto/bagisto#10178] [Q&A]
In product page when a user write his review his name is showing fully and for privacy most censor name characters like A**** G**** or user can select name view type.
Like screenshot 
![image](https://github.com/user-attachments/assets/d80897fe-a962-45a9-bd9b-fcd3a6529bff)

**Comment by amit-webkul** [comment]
Hello @aghaeian 

As per your suggestion regarding censoring reviewer names, we are discussing it with the team and will provide the best solution for this.

I also noticed that you have reported an issue for the same, so we will provide an update on that.

For now, I am closing this discussion. If you would like to discuss it further, feel free to reopen it.

**Reply by aghaeian** [reply]
Hello
Thank you for your support. 
Best Regards
